en/extend_your_application/README.md

```diff
@@ -181,12 +181,12 @@ Then, in a [PythonAnywhere Bash console](https://www.pythonanywhere.com/consoles
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd ~/<your-pythonanywhere-username>.pythonanywhere.com
+$ cd ~/<your-pythonanywhere-domain>.pythonanywhere.com
 $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+(Remember to substitute `<your-pythonanywhere-domain>` with your actual PythonAnywhere domain, without the angle-brackets.)
 
 
 ## Updating the static files on the server
```

Comment:
There's another occurrence [further down](https://github.com/DjangoGirls/tutorial/blame/4aa58695a0e7b0124c682e90c6413d72906153a9/en/extend_your_application/README.md#L200) in this section.

Reply:
I think I got it...